docs/topic-guides/blendable-web-apps/view-model-attaching.md

```diff
@@ -170,6 +170,12 @@ Blender.MapUri("/noparam1", token3, true, false);
 In the first example above, handler "/twoparams1/{?}/{?}" should not trigger calls on the `myToken` token, but it can be called when attaching is triggered by other handlers on the same token.  
 In the second example above, handler "/noparam1" can trigger other handlers on the same token, but can't be triggered by them.
 
+### Practical use
+
+If you have an app which specifies uni-directional mapping `Blender.MapUri(..., "settings", false ,true)`, then a Self.GET to your own URL will not return other views.
```

Comment:
Few corrections:

If you have an app which specifies a uni-directional rule `Blender.MapUri(..., "settings", false, true)`, then a `Self.GET` call to your own URL will not return other view-models.